src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
+        Machine[] machines = new Machine[]{excavator, bulldozer, truck};
```

Comment:
Make a note that you can call new Excavator(), new Bulldozer(), new Truck() explicitly in array initialization without temporary variables.

Reply:
> Make a note that you can call new Excavator(), new Bulldozer(), new Truck() explicitly in array initialization without temporary variables.

Thank you for your advice!
Added a new commit:
Moved call new Excavator(), new Bulldozer(), new Truck() in array initialization Machine[] machines in class MainApp